web-ng/src/app/services/data-store/data-store.service.ts

```diff
@@ -55,6 +55,41 @@ export class DataStoreService {
       );
   }
 
+  /**
+   * Returns an Observable that loads and emits the projects to which the user
+   * is part of.
+   *
+   */
+  loadAccessibleProject$(userEmail: string): Observable<List<Project>> {
+    return this.db
+      .collection('projects', ref =>
+        ref.where(new firebase.firestore.FieldPath('acl', userEmail), 'in', [
+          'owner',
+          'contributor',
+          'manager',
+          'viewer',
+        ])
+      )
+      .snapshotChanges()
+      .pipe(
+        map(projects =>
+          List(
+            projects
+              .filter(p => {
+                const docData = p.payload.doc.data() as DocumentData;
```

Comment:
You've already filtered the set of projects in the query above, so this filter is not necessary.

Reply:
removed.